packages/nextjs/app/apply/page.tsx

```diff
@@ -47,16 +49,48 @@ const Apply: NextPage = () => {
         ...fieldValues,
       });
       notification.success(`The grant proposal has been created`);
-      router.push("/my-grants");
+      feedbackModalRef.current?.showModal();
     } catch (error) {
       const errorMessage = getParsedError(error);
       notification.error(errorMessage);
     }
   };
 
+  const handleModalClose = () => {
+    feedbackModalRef.current?.close();
+    router.push("/my-grants");
+  };
+
   return (
     <div className="flex flex-col w-full items-center justify-center p-6 sm:p-10">
       <h2 className="text-2xl sm:text-3xl font-extrabold !mb-0">Apply for a grant</h2>
+      <p className="text-center text-lg text-base-content/80 mt-4 max-w-2xl">
+        Open to all public goods projects. ENS-specific projects should apply through the ENS Ecosystem Grants program
+        instead.
+      </p>
+
+      <dialog ref={feedbackModalRef} className="modal">
+        <div className="modal-box flex flex-col space-y-3">
+          <form method="dialog" className="bg-secondary -mx-6 -mt-6 px-6 py-4">
+            <div className="flex items-center">
+              <p className="font-bold text-xl m-0 text-center w-full">Application Submitted Successfully!</p>
+            </div>
+          </form>
+
+          <div className="text-center">
+            <p>
+              Grant applications are reviewed at a regular cadence, you will receive information if we need more info,
+              if the application was rejected and why, or approved and for what amount, based on the info submitted.
+            </p>
+            <div className="mt-6 flex justify-center">
+              <Button onClick={handleModalClose}>Accept</Button>
+            </div>
+          </div>
+        </div>
+        <form method="dialog" className="modal-backdrop">
```

Comment:
I think we don't need to close these two modals at backdrop click. User can misclick and then he'll not understand why he is on `/my-grants` page